TRANSLATIONS.md

```diff
@@ -27,17 +27,24 @@ GNU Gettext utilities below.
 
 ## Preparation
 
-We use two helpers for the translations:
+You will need the [Gettext] utilities (`msginit`, `msgmerge`). Under Ubuntu, you
```

Comment:
We use a Debian-based distribution at work, so I would like us to mention Debian here:

```suggestion
You will need the [Gettext] utilities (`msginit`, `msgmerge`). Under Debian and Ubuntu, you
```